clcache.py

```diff
@@ -7,6 +7,7 @@
 # root directory of this project.
 #
 from ctypes import windll, wintypes
+import cProfile
```

Comment:
:+1: 

It's 1-2 µs in a slow-io environment. The strange thing is that this is much faster than reading a single file – but well, Python seems to take care of that, which is great.